patches/chrome-installer-mac-sign_versioned_dir.sh.in.patch

```diff
@@ -26,19 +26,31 @@ index 7a853e3dcbffa5e7a218742ba55e4e629544d27b..434951f16964e2a11c4bccae87683b51
  notification_service="${framework}/XPCServices/AlertNotificationService.xpc"
  crashpad_handler="${framework}/Helpers/crashpad_handler"
  helper_app="${versioned_dir}/@MAC_PRODUCT_NAME@ Helper.app"
-@@ -119,18 +120,17 @@ codesign_with_options "${notification_service}" \
+@@ -115,22 +116,29 @@ xpc_plist="${notification_service}/Contents/Info"
+ xpc_bundle_id="$(__CFPREFERENCES_AVOID_DAEMON=1 defaults read \
+     "${xpc_plist}" "CFBundleIdentifier")"
+ 
++framework_plist="${framework}/Resources/Info"
++framework_bundle_id="$(__CFPREFERENCES_AVOID_DAEMON=1 defaults read \
```

Comment:
Fetch id from plist because framework of different channel have different id.
(ex, org.brave.Brave.dev.framework)

Reply:
isn't this also the case with chrome?

Reply:
it seems like we're patching some of these files to the point that we might as well have our own version of them?